### PR TITLE
[NFC] clarify per_ck

### PR DESCRIPTION
**Pull request by nkrackow**
closes https://github.com/quartiq/stabilizer/issues/647

### CODE REVIEW

File: src/hardware/setup.rs [hunk 1]
```diff
@@ -275,7 +275,7 @@ pub fn setup(
         .use_hse(8.MHz())
         .sysclk(design_parameters::SYSCLK.convert())
         .hclk(200.MHz())
-        .per_ck(design_parameters::TIMER_FREQUENCY.convert())
+        .per_ck(64.MHz()) // 64 MHz HSI clk selected per default (this setting changes nothing)
```
Comment (jordens) [inline src/hardware/setup.rs:278]:
```suggestion
        .per_ck(64.MHz()) // HSI, only used for internal ADC. This is not the "peripheral" clock for timers and others.
```

**Review by jordens — CHANGES_REQUESTED** [review]

File: src/hardware/setup.rs (hunk 1 above)
Comment (ryan-summers) [reply src/hardware/setup.rs:278]: Technically speaking, the `per_ck` _can_ be the peripheral "kernel" clock for some peripherals (i.e. SPI), but we don't use it as that: 
image

Comment (jordens) [reply src/hardware/setup.rs:278]: Yes. As the comment says, it is only used for ADC and not for anything else.

**Review by ryan-summers — APPROVED** [review]

**Pull request merged**